app/lib/shared/redis_cache.dart

```diff
@@ -274,6 +275,17 @@ class CachePatterns {
         encode: (bool value) => value,
         decode: (d) => d as bool,
       ))['$service/$package'];
+
+  /// Stores the Github OpenID JSON Web Key list.
+  Entry<JsonWebKeyList> githubOpenIdWebKeyList() => _cache
+      .withPrefix('github-openid-web-key-list/')
+      .withTTL(Duration(minutes: 15))
```

Comment:
Sad to see github responses doesn't carry a `Cache-Control` header allowing us to cache the response.
We should perhaps note that we are aware and decide to cache it for 15 minutes anyways. Just a comment.